src/app/shared/components/login-form/login-form.component.ts

```diff
@@ -8,5 +9,15 @@ import { NgForm } from '@angular/forms';
 })
 export class LoginFormComponent {
   @ViewChild('loginForm') public loginForm!: NgForm;
-  //Use the names `email` and `password` for form controls.
+  protected readonly ButtonText = ButtonText;
+  protected readonly IconNames = IconNames;
+
+  email = '';
```

Comment:
I's prefer to add the types for both email and password here - ngModel.

Reply:
I understand, but isn't it type-string automatically, because typescript recognize it from the default value which is a string?

In my IDE it looks like this btw. I tried to add the types manually but on save, it omits them. Maybe it is due to the nature of the IDE or/and my eslint/prettier setup, i don't know.
![login-component](https://github.com/user-attachments/assets/3b11a781-141c-44b2-a8fc-5c4773bc5300)

Reply:
Ohh, I thought LoginComponent was with the Reactive forms at first. No need to change anything for now.